### PR TITLE
Fix MLIP related issues with the benchmark results file

### PR DESCRIPTION
**Pull request by QuantumChemist**
also added the following commit :
- increase M3GNET RMSE tolereance because of fluctuations 
because of Merge pull request #238 from naik-aakash/combine_workflows · autoatml/autoplex@aea4326

TODOs:

- [x]  fix benchmark results file
- [x]  adjust the respective unit tests

### CODE REVIEW

**Comment by QuantumChemist** [comment]
@JaGeo my benchmark results file now looks like this
image

if this is ok, I only need to fix some more unit tests now.

**Comment by QuantumChemist** [comment]
@JaGeo , I have no idea what the jfr REMOTE_ERROR of the write_benchmark step could cause (the code looks fine to me), but I tried to change my jfr job retry settings and will see what happens, so maybe we can (temporary) fix the issue like that.

File: src/autoplex/data/phonons/flows.py [hunk 1]
```diff
@@ -393,7 +393,7 @@ def make(
             supercell_matrix_job = reduce_supercell_size_job(
                 structure=structure,
                 min_length=self.supercell_settings.get("min_length", 12),
-                max_length=self.supercell_settings.get("max_length", 25),
+                max_length=self.supercell_settings.get("max_length", 20),
```
Comment (QuantumChemist) [inline src/autoplex/data/phonons/flows.py:396]: @JaGeo this was why I had so large rattled supercells

Comment (JaGeo) [reply src/autoplex/data/phonons/flows.py:396]: But why not phonon ones?

Comment (QuantumChemist) [reply src/autoplex/data/phonons/flows.py:396]: There we set no max_length and I think it takes the fallback value in atomate2? I can't check the atomate2 code now.

Comment (JaGeo) [reply src/autoplex/data/phonons/flows.py:396]: I am confused. I thought we would use matrices for the supercells in both cases. At least, that is what i wanted to implement

Comment (QuantumChemist) [reply src/autoplex/data/phonons/flows.py:396]: You mean the same matrices? But it was implemented as separate ones. I can make it that it's the same matrix. Maybe in another PR?

Comment (JaGeo) [reply src/autoplex/data/phonons/flows.py:396]: There should be an option to make them the same.

**Comment by JaGeo** [comment]
Thanks @QuantumChemist :)

**Pull request merged**